src/api-reference/user-provisioning/spend/v4.spend-user-retrieval.md

```diff
@@ -217,58 +237,58 @@ GET https://www.us.api.concursolutions.com/spend/v4/Users/{uuid}
 
 #### Headers
 
-* `concur-correlationid` is a SAP Concur platform specific custom header used for technical support in the form of a [RFC 4122 A Universally Unique IDentifier (UUID) URN Namespace](https://tools.ietf.org/html/rfc4122)
+- `concur-correlationid` is a SAP Concur platform specific custom header used for technical support in the form of a [RFC 4122 A Universally Unique IDentifier (UUID) URN Namespace](https://tools.ietf.org/html/rfc4122)
 
 #### Payload
 
-* [-](#-)
+- [-](#-)
```

Comment:
Do we need to add anything to the section `Payload & Response`?

Reply:
The payload will be empty as its a GET but I can add a sample response.